torch/overrides.py

```diff
@@ -818,6 +832,12 @@ def get_testing_overrides() -> Dict[Callable, Callable]:
                                                                 distance_function=None, margin=1.0,
                                                                 swap=False, reduction='mean': -1),
         torch.nn.functional.unfold: lambda input, kernel_size, dilation=1, padding=0, stride=1: -1,
+        torch.nn.init.uniform_: lambda tensor, a=0., b=1.: -1,
+        torch.nn.init.constant_: lambda tensor, val: -1,
+        torch.nn.init.normal_: lambda tensor, mean=0., std=1.: -1,
+        torch.nn.init.constant_: lambda tensor, val: -1,
+        torch.nn.init.dirac_: lambda tensor, groups: -1,
```

Comment:
dirac_ doesn't seem to have a has_torch_function? https://github.com/pytorch/pytorch/blob/cc56fac2138c8a6504bea58a3f26d2d17b187c13/torch/nn/init.py#L256-L266

Reply:
looks wrong :)